scripts/src/lib/template/templates/gradle/kotlin/build.gradle.kts.eta

```diff
@@ -0,0 +1,113 @@
+plugins {
+	id("fabric-loom") version "1.0-SNAPSHOT"
+	id("maven-publish")<% if (it.kotlin) { %>
```

Comment:
```suggestion
	`maven-publish`<% if (it.kotlin) { %>
```
The standard Gradle plugins have extension properties that avoid the use of `id`. See Gradle's own example:
![](https://user-images.githubusercontent.com/6596629/210622356-4adbf0ad-e3b8-4712-b4ff-bb783444597f.png)